Formula/amp.rb

```diff
@@ -0,0 +1,48 @@
+class Amp < Formula
+  desc "Text editor for your terminal"
+  homepage "https://amp.rs"
+  url "https://github.com/jmacdonald/amp/archive/0.3.2.tar.gz"
+  sha256 "9080c894111149d58a0b805e0c08edaf1d9e7cd56c4b955ba9111feec6b55208"
+  head "https://github.com/jmacdonald/amp.git"
+
+  depends_on "rust" => :build
+  depends_on "cmake" => :build
+  depends_on "openssl"
+  depends_on "libssh2"
+
+  def install
+    ENV["OPENSSL_INCLUDE_DIR"] = Formula["openssl"].opt_include
+    ENV["DEP_OPENSSL_INCLUDE"] = Formula["openssl"].opt_include
+    ENV["OPENSSL_LIB_DIR"] = Formula["openssl"].opt_lib
+    ENV.prepend "LDFLAGS", "-L#{Formula["openssl"].opt_lib}"
```

Comment:
You can drop all of these beside `ENV["OPENSSL_INCLUDE_DIR"]` and `ENV["OPENSSL_LIB_DIR"]`.

Reply:
Done. Is the `PKG_CONFIG_PATH` still needed?

Reply:
Nope, apparently not.